packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -169,10 +169,18 @@
 	top: $grid-unit;
 	opacity: 0;
 
+	&.edit-site-global-styles__shadow-editor__remove-button {
+		border: none;
+	}
```

Comment:
I needed to increase the specificity of the style to reset the border, otherwise the border would remain below the button in the mobile layout.

![image](https://github.com/user-attachments/assets/f76cf3ef-daa2-4be0-bb6e-011472c0dfa7)